### PR TITLE
OMERO 5.4.5 release

### PR DESCRIPTION
**Pull request by hflynn**
See https://trello.com/c/Nbu1wFxV/40-open-pr-against-wwwopenmicroscopyorg

This uses the text from https://github.com/openmicroscopy/openmicroscopy/pull/5687

### CODE REVIEW

**Comment by hflynn** [comment]
Staged at https://snoopycrimecop.github.io/www.openmicroscopy.org/2018/03/21/omero-5-4-5.html and https://snoopycrimecop.github.io/www.openmicroscopy.org/omero/downloads/

**Comment by mtbc** [comment]
I'm guessing we don't yet expect the artifacts to be downloadable?

**Comment by hflynn** [comment]
That's correct @mtbc - that needs the release build to have been run

File: _posts/2018-03-21-omero-5-4-5.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+* fixed display of thumbnails when searching for images by ID
+* increased OMERO.web log size
+* fixed CLI list subcommand
+
+This release does not upgrade the version of Bio-Formats which OMERO uses,
+which remains at 5.7.3.
+
+The software is also available at [archived downloads](https://downloads.openmicroscopy.org/omero/5.4.5).
+
+Any problems or comments, please use the [OME Forums or mailing lists]({{ site.baseurl }}/support/)
```
Comment (mtbc) [inline _posts/2018-03-21-omero-5-4-5.md]: Weird this doesn't end with a period when the sentences above do.

File: _posts/2018-03-21-omero-5-4-5.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+Today we are releasing OMERO 5.4.5, a bug-fix release reactivating the
+thumbnail cache inadvertently disabled in 5.4.4 while fixing a pyramid issue.
+
+Improvements include:
+
+* reactivated thumbnail caching
+* improved removepyramids help
+* fixed display of thumbnails when searching for images by ID
+* increased OMERO.web log size
+* fixed CLI list subcommand
```
Comment (mtbc) [inline _posts/2018-03-21-omero-5-4-5.md]: "fixed CLI config list subcommand"

**Comment by hflynn** [comment]
Latest commit adds extra entry from https://github.com/openmicroscopy/openmicroscopy/pull/5694/

**Comment by jburel** [comment]
now staged at https://snoopycrimecop.github.io/www.openmicroscopy.org/2018/03/28/omero-5-4-5.html

**Comment by jburel** [comment]
build number and version are correct

**Comment by jburel** [comment]
Tested the link now the artifacts are available.
Everything works
Merging

**Pull request merged**